src/chainlit/element.py

```diff
@@ -1,97 +1,75 @@
 from pydantic.dataclasses import dataclass
 from dataclasses_json import dataclass_json
-from typing import Dict
+from typing import Dict, Union
 import uuid
-from abc import ABC, abstractmethod
-from chainlit.sdk import get_sdk, BaseClient
+import aiofiles
+
+from chainlit.emitter import get_emitter, BaseClient
 from chainlit.telemetry import trace_event
 from chainlit.types import ElementType, ElementDisplay, ElementSize
-from base64 import b64encode, b64decode
-from urllib import parse as urlparse
+
+type_to_mime = {
+    "image": "image/png",
```

Comment:
I would prefer to stick with a generic mime type here, to prevent any issue with an incorrect mime type.
The suggested mime type is the one we attribute to non-pdf elements, so it looks like the best fit.

```suggestion
    "image": "binary/octet-stream",
```